package.json

```
@@ -55,7 +55,7 @@
     "prettier": "^2.8.8"
   },
   "engines": {
-    "node": "^16 || ^18",
+    "node": "^16 || ^18 || ^20",
```

I'd change it to
```suggestion
    "node": ">=18 <=21",
```

since https://nodejs.org/en/blog/announcements/nodejs16-eol

---

Arguably it could just be 18+ (or 16+, if there are lots of legacy installs).

Future versions are pretty stable in terms of API, and handling version bump PRs every 6 months could be avoided...